cross/libarchive/PLIST

```diff
@@ -0,0 +1,3 @@
+lib:lib/libarchive.so
+lib:lib/libarchive.so.13
```

Comment:
Those are links to `libarchive.so.13.7.3`. If prefixed with `lib` the package will contain duplicates instead of links.
```suggestion
lnk:lib/libarchive.so
lnk:lib/libarchive.so.13
```

Reply:
Aha! I noticed this in the output tarball, but didn't understand at the time. Now I do, thx!